backend/accounts.go

```diff
@@ -73,6 +73,30 @@ func (a AccountsList) lookup(code accountsTypes.Code) accounts.Interface {
 	return nil
 }
 
+// lookupByTransactionInternalID finds the account which contains a transaction with this internal
+// tx ID. `nil, nil` is returned if not found. `err` is returned if there was an error fetching the
+// account transactions.
+func (a AccountsList) lookupByTransactionInternalID(internalID string) (accounts.Interface, error) {
```

Comment:
I think I didn't because `TestNotesImportWithoutBitBoxAppData()` covers these lines as well and I'd have to re-do most of `func (s *notesTestSuite) SetupTest() {}` to set it up :thinking: not sure if worth it